### PR TITLE
scripts: add attached options for kconfig

### PR DESCRIPTION
**Pull request by xiulipan**
We can attach any debug kconfig or board specific defconfig with the
scripts.

EG: ../kconfig/kconfig-sof-default.sh ../kconfig/debug-defconfig

Signed-off-by: Pan Xiuli <xiuli.pan@linux.intel.com>

### CODE REVIEW

**Comment by xiulipan** [comment]
@plbossart I think this will extend the usage of our scripts.

File: kconfig-hda.sh [hunk 1]
```diff
@@ -2,4 +2,4 @@
 KCONFIG_DIR=$(dirname ${BASH_SOURCE[0]})
 echo $KCONFIG_DIR
 make defconfig
-scripts/kconfig/merge_config.sh .config $KCONFIG_DIR/base-defconfig $KCONFIG_DIR/hdaudio-codecs-defconfig
+scripts/kconfig/merge_config.sh .config $KCONFIG_DIR/base-defconfig $KCONFIG_DIR/hdaudio-codecs-defconfig $@
```
Comment (plbossart) [inline kconfig-hda.sh]: can we generalize the idea and have additional arguments, e.g. to disable PM and enable debug?

Comment (xiulipan) [reply kconfig-hda.sh]: @plbossart this feature will provide the feature to add additional kconfig to base kconfig by the user. They can try to use like `../kconfig/kconfig-sof-default.sh ../kconfig/nopm-defconfig` if they would like a sof-default kconfig with nopm to do some test.

Comment (plbossart) [reply kconfig-hda.sh]: right, but what I meant is that $@ means 'all the arguments', so you could have more than one kconfig passed as argument

**Pull request merged**